Some updates to 2.1.50 (#109)
ankitects/anki-manual: src/browsing.md

| Name | Action |
|-|-|
| Add Notes | Open the [Add](editing.md#adding-cards-and-notes) dialog. |

> | Create Copy | Open a [duplicate](browsing.md#finding-duplicates) of the current note in the [editor](editing.md#adding-cards-and-notes), which can be slightly modified to easily obtain variations of your cards. By default, the duplicate card will be created in the same deck that the original. |

Reviewer: ```suggestion
| Create Copy | Open a [duplicate](browsing.md#finding-duplicates) of the current note in the [editor](editing.md#adding-cards-and-notes), which can be slightly modified to easily obtain variations of your cards. By default, the duplicate card will be created in the same deck as the original. |
```

|-|-|
| Change Deck | Move currently selected cards to a different deck. |
| Set Due Date | Turn cards into review cards, and make them due on a certain date. This can be useful for moving cards forward or back a few days when your study schedule is interrupted. Entering a range like `60-90` will make the selected cards due between 60 and 90 days from now. New cards will have their interval set to the same delay, but reviews will be rescheduled without changing their current interval, unless '!' is included at the end of the range.

> | Forget | Move currently selected cards to the end of the new queue. The existing review history is preserved. In 2.1.50+, you have the options to restore the original card position, and to reset the card's lapse and repetition counters.|

Reviewer: ```suggestion
| Forget | Move currently selected cards to the end of the new queue. The existing review history is preserved. In 2.1.50+, there are options to restore the original card position, and to reset the card's lapse and repetition counters.|
```
